src/components/structured-list/README.md

```diff
@@ -0,0 +1,37 @@
+### SCSS
+
+#### Modifiers
+
+Use these modifiers with `.bx--structured-list` class.
+
+| Selector                        | Description                                |
+|---------------------------------|--------------------------------------------|
+| .bx--structured-list--border    | Applies border around structured-list      |
+| .bx--structured-list--condensed | Applies condensed styles for all body rows |
+| .bx--structured-list-content--nowrap | Applies `white-space: nowrap;` on content |
```

Comment:
Todo: 
- [ ] nowrap is to prevent title cells from wrapping. For example: "BigInsights for Hadoop" will stay on a single line. 
- [ ] clean up table markdown code